.cirrus.yml

```diff
@@ -135,3 +135,17 @@ task:
     image: ubuntu:bionic
   env:
     FILE_ENV: "./ci/test/00_setup_env_mac.sh"
+
+task:
+  name: 'macOS 10.14 native [GOAL: install] [GUI] [no depends]'
+  macos_brew_addon_script:
+    - brew install boost libevent berkeley-db4 qt miniupnpc qrencode sqlite ccache zeromq qrencode sqlite libtool automake pkg-config coreutils gnu-getopt
```

Comment:
fafc0158f99d90078223461631ef2829bd5fcec6

The `qrencode` and `sqlite` packages are mentioned twice :)

Why the `coreutils` and `gnu-getopt` packages are required (comparing to [doc/build-osx.md](https://github.com/bitcoin/bitcoin/blob/master/doc/build-osx.md))?

Reply:
thanks removed duplicate packets. 

getopt is needed for `IN_GETOPT_BIN="/usr/local/opt/gnu-getopt/bin/getopt"` in the same commit